include/lsqecc/gates/gates.hpp

```diff
@@ -35,7 +35,9 @@ struct BasicSingleQubitGate{
         S,
         T,
         H,
-
+        
+        SDg,
+        TDg,
```

Comment:
You can add `bool is_inverse = false` below.

Then whether we don't even need to keep the `*Dg`.

Reply:
I meant it's up to you :)